src/RequestHeaders.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestProvider;
+
+final class RequestHeaders {
```

Comment:
- Add phpdoc descriptions to class and methods
- Add docs in readme
- Add line to changelog